pulp_rpm/app/models/advisory.py

```diff
@@ -236,8 +236,8 @@ class UpdateCollection(BaseModel):
         update_record (models.ForeignKey): The associated UpdateRecord
     """
 
-    name = models.TextField()
```

Comment:
Why nullable?  See below:

Reply:
opensuse doesn't use it so can be null not only empty. same as shortname
can be seen in update.xml in repo mentioned above

Reply:
How safe would it be to collapse it into an emptystring regardless?

Reply:
If it will be an empty string createrepo_c will generate empty element to xml back when publish/distribute, which I think is unnecessary